src/components/05_pages/NodeForm/RecipeSchema.js

```diff
@@ -133,8 +133,8 @@ export default {
         field_difficulty: {
           type: 'string',
           title: 'Difficulty',
-          enum: ['easy', 'medium', 'hard'],
-          default: 'medium',
+          options: ['easy', 'medium', 'hard'],
+          defaultValue: 'medium',
```

Comment:
Change values, as ones used are reserved, so using them is not possible when deconstructing objects.

Reply:
Mh, I fear we will not be able to change this output. This is a copy of what will come back from openapi.

Reply:
Change made and I also updated the component.